server/lessons/el/analysh-dedomenwn-twitter.html

```diff
@@ -118,8 +118,8 @@ <h4>Επόμενο</h4>
     <p>Αποκτήστε τον κώδικα ενσωμάτωσης για να δημοσιεύσετε το γράφημα κάνοντας κλικ στο κουμπί <img class="icon-insert" src="./embed_button.svg" /> στο επάνω δεξιό μέρος του πίνακα γραφημάτων.</p>
     <h4>Συνεχίστε τα μαθήματα:</h4>
     <ol>
-      <li><a href="/lessons/el/prosarmosmenes-eidopoihseis/">Παρακολουθήστε δεδομένα και ορίστε προσαρμοσμένες ειδοποιήσεις</a></li>
-      <li><a href="/lessons/el/kathariste-vromika-dedomena/">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
+      <li><a href="prosarmosmenes-eidopoihseis">Παρακολουθήστε δεδομένα και ορίστε προσαρμοσμένες ειδοποιήσεις</a></li>
+      <li><a href="kathariste-vromika-dedomena">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
```

Comment:
The text in `<a href="kathariste-vromika-dedomena"></a>` should be "Καθαρίστε και τυποποιήστε δεδομένα"